code/modules/mob/living/carbon/human/species_types/abductors.dm

```diff
@@ -9,8 +9,10 @@
 		TRAIT_VIRUSIMMUNE,
 		TRAIT_NOBLOOD,
 		TRAIT_CHUNKYFINGERS_IGNORE_BATON,
+		/* Bubber Edit - Abductor Nerf
 		TRAIT_NODISMEMBER,
 		TRAIT_NEVER_WOUNDED
+		*/
 	)
```

Comment:
Pretty sure this messes up with the actual abductors. Could you maybe make it its own inherited species datum and make this one not available for crew.